ansible_ai_connect_admin_portal/public/static/console/static/locales/en/translation.json

```diff
@@ -10,9 +10,11 @@
   "APIKey": "IBM Cloud API Key",
   "APIKeyTooltip": "Used to authenticate with IBM watsonx Code Assistant.",
   "APIKeyDescription": "The API Key for IBM watsonx Code Assistant.",
+  "APIKeyDeletionConfirmation": "Are you sure you want to delete your IBM Cloud API Key?",
```

Comment:
I am going to ask about this text to Marty once I start the backend part. For now since we dont see it, pls ignore the product owner approval. I will get the related approvals.